test/deploy/kubernetes/manual/deploy.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+SCRIPT_DIR=$(dirname $0)
+MAXNODES="3"
+NAMESPACE="cortx"
+
+function print_header {
+    echo -e "---------------------------------------------------"
+    echo -e "$1"
+    echo -e "---------------------------------------------------"
+}
+
+# Create NameSpace
+kubectl create namespace $NAMESPACE
+
+# Create ConfigMap
+kubectl create configmap solution-config \
+    --from-file=$SCRIPT_DIR/solution-config/cluster.yaml \
+    --from-file=$SCRIPT_DIR/solution-config/config.yaml \
+    --namespace $NAMESPACE
+
+# Create Persistent Volumes for Block Devices
+for NODE_INDEX in $(seq 1 $MAXNODES); do
+    NODE_NAME="node"$NODE_INDEX;
+    print_header "Creating Persistent Volumes - $NODE_NAME"
+    NODE_PVOL=$SCRIPT_DIR/persistent-volumes/block_devices_$NODE_NAME.yaml;
+    kubectl apply -f $NODE_PVOL --namespace $NAMESPACE;
+    sleep 2;
+    kubectl get pv --namespace $NAMESPACE | grep $NODE_NAME;
+done
+
+# Create Persistent Volume Claims for Block Devices
+for NODE_INDEX in $(seq 1 $MAXNODES); do
+    NODE_NAME="node"$NODE_INDEX;
+    print_header "Creating Persistent Volume Claims - $NODE_NAME"
+    NODE_PVCS=$SCRIPT_DIR/volume-claims/blockdevices_claim_$NODE_NAME.yaml;
+    kubectl apply -f $NODE_PVCS --namespace $NAMESPACE;
+    sleep 2;
+done
+
+# Create Persistent Volumes for Block Volumes
+for NODE_INDEX in $(seq 1 $MAXNODES); do
+    NODE_NAME="node"$NODE_INDEX;
+    print_header "Creating Persistent Volumes - $NODE_NAME"
+    NODE_BVOL=$SCRIPT_DIR/persistent-volumes/block_volumes_$NODE_NAME.yaml;
+    kubectl apply -f $NODE_BVOL --namespace $NAMESPACE;
+    sleep 2;
+    kubectl get pv --namespace $NAMESPACE | grep $NODE_NAME;
+done
+
+# Create Persistent Volume Claims for Block Volumes
+for NODE_INDEX in $(seq 1 $MAXNODES); do
+    NODE_NAME="node"$NODE_INDEX;
+    print_header "Creating Persistent Volume Claims - $NODE_NAME"
```

Comment:
print_header "Creating Persistent Volume Claims for Storage Devices - $NODE_NAME"